packages/travix-ui-kit/package.json

```diff
@@ -4,23 +4,21 @@
   "description": "Travix UI kit",
   "main": "lib/index.js",
   "scripts": {
-    "build:watch": "node ./builder/program -w",
+    "build:watch": "npm run build -- -w",
     "build": "node ./builder/program",
-    "ci:comment-url": "curl https://api.github.com/repos/$TRAVIS_REPO_SLUG/issues/$TRAVIS_PULL_REQUEST/comments -u \"$TRAVIX_SERVICE_ACCOUNT:$TRAVIX_SERVICE_ACCOUNT_TOKEN\" --data \"{ \\\"body\\\": \\\"A new acceptance URL has been deployed: [$ACCP_URL]($ACCP_URL)\\\" }\"",
-    "ci:lint": "eslint --color --max-warnings 0 --quiet '{components,tests,utils,scripts}/**/*.js'",
+    "ci:lint": "npm run lint -- --max-warnings 0 --quiet",
     "ci:send-coverage": "codecov",
-    "ci:test": "TZ=utc jest -c ./tests/jest.config.json --bail --ci --coverage --maxWorkers=2 --no-cache --silent",
-    "deploy:accp": "BRANCH=$(git rev-parse --abbrev-ref HEAD); make prepare-site && now rm -y $BRANCH 2>/dev/null; now _site --public -n $BRANCH",
-    "deploy:accp:ci": "now -t \"${NOW_SH_AUTH_TOKEN}\" rm -y \"${TRAVIS_PULL_REQUEST_BRANCH}\" 2>/dev/null; now -t \"${NOW_SH_AUTH_TOKEN}\" _site --public -n \"${TRAVIS_PULL_REQUEST_BRANCH}\"",
+    "ci:test": "npm test -- --bail --ci --coverage --maxWorkers=2 --no-cache --silent",
     "lint": "eslint --color '{components,tests,utils,scripts}/**/*.js'",
-    "prebuild:watch": "babel --copy-files ./components --out-dir lib --ignore *.scss,*.md -w &",
-    "prebuild": "babel --copy-files ./components --out-dir lib --ignore *.scss,*.md &",
+    "prebuild:watch": "npm run transpile -- -w &",
+    "prebuild": "npm run transpile -- &",
+    "prestyleguide:build": "npm run build",
     "styleguide:build": "styleguidist build",
     "styleguide:server": "styleguidist server",
     "styleguide:dev": "npm run styleguide:build && npm run styleguide:server",
     "test": "TZ=utc jest -c ./tests/jest.config.json",
-    "transpile": "npm run build",
-    "update-snapshots": "TZ=utc jest -c ./tests/jest.config.json -u"
+    "transpile": "babel --copy-files ./components --out-dir lib --ignore *.scss,*.md",
+    "update-snapshots": "npm test -- -u"
```

Comment:
I see you are removing the definition of timezone usage `TZ=utc` from the test commands. Did you confirm this is not needed anymore? We added this in the past since we had failing unit test with the Calendar component, where the snapshots would not match (If i remember correctly).

Reply:
I haven't removed `TZ=utc` from test commands—it's still part of `npm test`, which is being reused in `ci:test` and `update-snapshots`, so the final expanded commands remain the same. For example:

```sh
$ npm run update-snapshots

> travix-ui-kit@0.10.1 update-snapshots ui/packages/travix-ui-kit
> npm test -- -u

> travix-ui-kit@0.10.1 test ui/packages/travix-ui-kit
> TZ=utc jest -c ./tests/jest.config.json "-u"
```